### PR TITLE
ADEN-5173 Get rid of INCONTENT_LEADERBOARD

### PR DESCRIPTION
**Pull request by fraszczakszymon**
and remove geo restriction from INCONTENT_PLAYER

### CODE REVIEW

File: extensions/wikia/AdEngine/js/context/slotsContext.js [hunk 1]
```diff
@@ -30,8 +30,7 @@ define('ext.wikia.adEngine.context.slotsContext', [
 		setStatus('INCONTENT_BOXAD_1', !isHome);
 
 		setStatus('INVISIBLE_HIGH_IMPACT_2', geo.isProperGeo(instantGlobals.wgAdDriverHighImpact2SlotCountries));
-		setStatus('INCONTENT_LEADERBOARD', geo.isProperGeo(instantGlobals.wgAdDriverIncontentLeaderboardSlotCountries));
-		setStatus('INCONTENT_PLAYER', geo.isProperGeo(instantGlobals.wgAdDriverIncontentPlayerSlotCountries));
+		setStatus('INCONTENT_PLAYER', true);
```
Comment (dianafa) [inline extensions/wikia/AdEngine/js/context/slotsContext.js]: what about moving it to line 26, to "slots that exists on all pages" section?

File: extensions/wikia/AdEngine/js/spec/context/slotsContext.spec.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 
 /*global beforeEach, describe, it, expect, modules, spyOn*/
-describe('ext.wikia.adEngine.context.uapContext', function () {
+describe('ext.wikia.adEngine.context.slotsContext', function () {
```
Comment (dianafa) [inline extensions/wikia/AdEngine/js/spec/context/slotsContext.spec.js:3]: 👍

**Comment by dianafa** [comment]
LGTM

File: extensions/wikia/AdEngine/js/run/desktop.run.js [hunk 3]
```diff
@@ -98,48 +98,33 @@ require([
...
-	'ext.wikia.adEngine.adContext',
 	'ext.wikia.adEngine.context.slotsContext',
 	'ext.wikia.adEngine.slot.bottomLeaderboard',
 	'ext.wikia.adEngine.slot.highImpact',
 	'ext.wikia.adEngine.slot.inContent',
 	'ext.wikia.adEngine.slot.skyScraper3',
-	'ext.wikia.adEngine.slotTweaker',
 	'wikia.document',
 	'wikia.window'
 ], function (
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/run/desktop.run.js:108]: MAJOR Function has 7 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 32 issues
* ![MAJOR][MAJOR] 31 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L23](https://github.com/Wikia/app/blob/5ef439f765b58c0c34e4c49fbe55bd41a7ab8333/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L23): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L23](https://github.com/Wikia/app/blob/5ef439f765b58c0c34e4c49fbe55bd41a7ab8333/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L23): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L23](https://github.com/Wikia/app/blob/5ef439f765b58c0c34e4c49fbe55bd41a7ab8333/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L23): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L23](https://github.com/Wikia/app/blob/5ef439f765b58c0c34e4c49fbe55bd41a7ab8333/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L23): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L23](https://github.com/Wikia/app/blob/5ef439f765b58c0c34e4c49fbe55bd41a7ab8333/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L23): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L98](https://github.com/Wikia/app/blob/5ef439f765b58c0c34e4c49fbe55bd41a7ab8333/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L98): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L99](https://github.com/Wikia/app/blob/5ef439f765b58c0c34e4c49fbe55bd41a7ab8333/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L99): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L199](https://github.com/Wikia/app/blob/5ef439f765b58c0c34e4c49fbe55bd41a7ab8333/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L199): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L224](https://github.com/Wikia/app/blob/5ef439f765b58c0c34e4c49fbe55bd41a7ab8333/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L224): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L224](https://github.com/Wikia/app/blob/5ef439f765b58c0c34e4c49fbe55bd41a7ab8333/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L224): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**